src/main/java/com/redhat/labs/lodestar/engagements/model/Engagement.java

```diff
@@ -139,7 +139,7 @@ public boolean overrideImmutableFields(Engagement stone, boolean categoryUpdate)
             projectId = stone.getProjectId();
         }
         
-        if(stone.getLaunch() != null) { //Launch has already happened
+        if(stone.getLaunch() != null && launch != null) { //Launch has already happened - you can delete but not change the launch info
```

Comment:
Does the comment imply that the actual information in the `launch: {}` cannot be changed? As in the past, this would be useful to be able to change as well. I'd actually recommend that we don't think of this is as a "delete", but rather that the .json data can be changed.